cli/internal/opgraph/callgraph.go

```diff
@@ -37,6 +37,9 @@ var errNotFoundInGraph = errors.New("not found in graph")
 const skippedState = "skipped"
 
 func (n *callGraphNode) insert(call *model.Call, startTime time.Time, initialState string) error {
+	if n == nil || n.call == nil || call == nil {
```

Comment:
@zachpuck this was the nil pointer bug we saw sometimes; it's got to do with event order not being guaranteed so n can be nil.